packages/dd-trace/src/noop/proxy.js

```diff
@@ -1,12 +1,14 @@
 'use strict'
 
 const NoopTracer = require('./tracer')
+const NoopAppsecSdk = require('../appsec/sdk/noop')
 
 const noop = new NoopTracer()
 
 class Tracer {
   constructor () {
     this._tracer = noop
+    this.appsec = new NoopAppsecSdk()
```

Comment:
Consistency with the NoopTracer, it's a singleton
```suggestion
const noop = new NoopTracer()
const noopAppsec = new NoopAppsecSdk()

class Tracer {
  constructor () {
    this._tracer = noop
    this.appsec = noopAppsec
```